src/shared/auth/auth-flow.ts

```diff
@@ -32,6 +33,8 @@ import {
 
 const requestor = new FetchRequestor();
 
+const TOKEN_RESPONSE_NAME = 'umb:tokenResponse';
```

Comment:
Could we provide a name here that is a bit more specific, like userAuthTokenResponse ?

Reply:
Done